ibexa/commerce/4.6/config/routes/payum.yaml

```diff
@@ -0,0 +1,2 @@
+payum_all:
+    resource: "@PayumBundle/Resources/config/routing/all.xml"
```

Comment:
At first I thought we shouldn't add it because it's should be added by PayumBundle recipe. Checked and they are not adding their routing in the recipe 🤔